syntax = "proto3";

package suggest;

message Book {
    string bookId = 1;
    string title = 2;
    string author = 3;
}


service SuggestionsService {
    rpc Suggestions (SuggestionsRequest) returns (SuggestionsResponse);
}

message SuggestionsRequest {
    repeated Book user_books = 1;
}

message SuggestionsResponse {
    repeated Book suggested_books = 1;
}
